Climate conditions: target (Labs feature) (#45439)
home-assistant/home-assistant.github.io: source/_conditions/climate.target_temperature.markdown

---
title: "Thermostat target temperature"
condition: climate.target_temperature
domain: climate
description: "Tests the target temperature of one or more thermostats."
related_conditions:
  - climate.is_on
  - climate.is_cooling
  - climate.is_heating
  - climate.is_drying

>   - climate.is_hvac_mode

Author: they will be there shortly

  - climate.target_humidity
---

> The **Thermostat target temperature** condition passes when a thermostat {% term entity %}'s target temperature setting meets a threshold you define. The target temperature is the setpoint you configure on the device, not the actual current temperature reading. For example, you can use it to send a notification when the thermostat is set to a temperature that's unusually high or low.

Reviewer: Same as above.

> The **Thermostat target temperature** condition passes when a thermostat {% term entity %}'s target temperature setting meets a threshold you define. The target temperature is the setpoint you configure on the device, not the actual current temperature reading. For example, you can use it to send a notification when the thermostat is set to a temperature that's unusually high or low.

Author: ```suggestion
The **Thermostat target temperature** condition passes when a thermostat {% term entity %}'s target temperature setting meets a threshold you define. The target temperature is the setpoint you configure on the device, not the actual current temperature reading. For example, you can use this condition in an automation to turn on a dehumidifier only if the thermostat is set to an unusually high or low temperature.
```
